products/nuxt.md

```diff
@@ -39,5 +39,5 @@ year, with an expectation of patch releases every week or so and minor releases
 Minor and Patch releases should never contain breaking changes except for features marked as
 _experimental_.
 
-Nuxt 2 will reach End of Life (EOL) on December 31st, 2023 [at the same time as Vue 2 does](/vue).
+Nuxt 2 will reach End of Life (EOL) on June 30th, 2024.
```

Comment:
```suggestion
```

Now that the information about Vue 2 as been removed I see no point of keeping this sentence (That's what is already documented above).